index.html

```diff
@@ -29,18 +42,20 @@
             </li>
           </ul>
         </div>
+        
 
-        <nav class="nav-link">
-          <div class="nav-menu">
-            <div class="ambuger-menu">
-              <span class="bar"></span>
-              <span class="bar"></span>
-              <span class="bar"></span>
+          <nav class="nav-link">
+            
+            <div class="nav-menu">
+              
+              <div class="ambuger-menu">
+               <a href="#" onclick="openNav()"> <img src="/images/icon-menu.png" alt="" srcset=""></a>
+              </div>
```

Comment:
- [ ] after making some changes on my side I was able to test the menu itself and it works great, however, the "hamburger-button" is still not visible, please make sure it is visible before requesting another review.